packages/react-components/src/molecules/SearchField.tsx

```diff
@@ -1,18 +1,31 @@
 import React, { ComponentProps } from 'react';
+import css from '@emotion/css';
 
 import { TextField } from '../atoms';
 import { searchIcon } from '../icons';
+import { lead, pine } from '../colors';
+
+const containerStyles = css({
```

Comment:
I think this was better placed in `TextField` than `SearchField`

* would need to know `TextField`s internal structure, and
* any `TextField` consumer can set a custom indicator, and the indicator will need to respond to `:focus`

Reply:
I put it here because some of the indicators (Show and Hide) need to not respond to :focus

Reply:
Right, in fact regardless on focus they must not have `stroke`. Hmm ...

Reply:
revert this and add `stroke="none"` to the show and hide `<text>` elements? :thinking: not super sleek...